src/controllers/searchDoc/searchFoundItem.js

```diff
@@ -0,0 +1,37 @@
+import Document from '../../models/lost';
+
+class SearchController {
+  static async searchFoundItem(req, res) {
+    try {
+      const foundDocs = await Document.find({
+        $and: [
+          { documentName: req.body.documentName },
+          { 'status.isFound': true }
+        ]
+      });
+      if (foundDocs.length === 0) return res.status(400).json({ msg: 'No such document in database' });
+
+      return res.status(200).send({ foundDocs });
+    } catch (err) {
+      return res.status(400).send({ error: err.message });
+    }
+  }
+
+  static async searchLostItem(req, res) {
+    try {
+      const lostDocs = await Document.find({
+        $and: [
+          { documentName: req.body.documentName },
+          { 'status.isLost': true }
```

Comment:
The same here, are we searching based on just document name, we can search for all items matching a particular text and on all fields.

Reply:
I thought of being specific but we can go for full text search to display matching items.